.ci_build_samples.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/sh
 set -e
 
+export NXDK_DIR=`pwd`
+export PATH="${PATH}:${NXDK_DIR}/usr/bin:${NXDK_DIR}/usr/local/bin"
```

Comment:
This might pose problems because this could install `sdl-config` etc.
Those tools will need a prefix.